src/pages/patientView/mutation/column/AlleleFreqColumnFormatter.tsx

```diff
@@ -7,17 +7,21 @@ import {compareNumberLists} from '../../../../shared/lib/SortUtils';
 import 'rc-tooltip/assets/bootstrap_white.css';
 import {MutationTableRowData} from "../../../../shared/components/mutationTable/IMutationTableProps";
 import {Mutation} from "../../../../shared/api/generated/CBioPortalAPI";
+import SampleManager from "../../sampleManager";
 
 export default class AlleleFreqColumnFormatter {
     static barWidth = 6;
     static barSpacing = 3;
     static maxBarHeight = 12;
     static indexToBarLeft = (n:number) => n*(AlleleFreqColumnFormatter.barWidth + AlleleFreqColumnFormatter.barSpacing);
 
-    public static renderFunction(data:IColumnFormatterData<MutationTableRowData>, columnProps:any) {
-        const sampleOrder = columnProps.sampleManager.getSampleIdsInOrder();
+    public static renderFunction(mutations:Mutation[], sampleManager:SampleManager|null) {
+        if (!sampleManager) {
```

Comment:
And TS complains if you don't allow for null?  b/c i think by definition it should not be null when table renders because there's a check for sampleManager around the table.